f5cd2de

commit f5cd2de
Author: Alain Delree <alain.delree@gmail.com>
Date:   Mon Aug 3 16:24:58 2026 +0200

    Publication v2 : chemin de config relatif à l'exécutable (multi-instances)

diff --git a/version.json b/version.json
index 0c05d7a..5a66145 100644
--- a/version.json
+++ b/version.json
@@ -1 +1 @@
-{"build": 1, "sha256": "adcef67c4c66779a4e09df4c43c15124476b66411d05e1b1847c10b772f06ac8"}
+{"build": 2, "sha256": "4ea166fd661df65147ca53f11b6d6ff466f850a2a33a279381db3de9ca6f1cc9"}
